API_REFERENCE.md

```diff
@@ -68,6 +68,12 @@ The entry point to use Datadog's Trace feature.
 
 The entry point to use Datadog's RUM feature.
 
+- `setContext(context: map)`
+
+    Sets the global RUM context (set of attributes) attachd with all future RUM events.
```

Comment:
In native SDKs we widely use term of "global attributes" (for `Logger`, `Tracer` and `RUMMonitor`). Cannot we apply it to RN SDK as well? I believe that unified lingo may help us with maintaining concepts across the entire stack.
```js
- `setAttributes(attributes: map)`
```

Reply:
It does make sense ;)